src/Symfony/Component/Mailer/composer.json

```diff
@@ -17,7 +17,7 @@
     ],
     "require": {
         "php": ">=7.2.5",
-        "egulias/email-validator": "^2.1.10",
+        "egulias/email-validator": "^2.1.10|^3.0",
```

Comment:
There are more places where this package needs to be bumped. Furthermore, we should try to patch the 4.4 branch instead.

Reply:
yes, on all the mailer components

Reply:
I would prefer 4.4, too